pandas/core/arrays/base.py

```diff
@@ -862,6 +863,27 @@ def copy(self) -> ABCExtensionArray:
         """
         raise AbstractMethodError(self)
 
+    def view(self, dtype=None) -> Union[ABCExtensionArray, np.ndarray]:
```

Comment:
why would this return a np.ndarray OR an EA for an EA? (is this @jorisvandenbossche question)?

when / why would this be the case? this is pretty confusing.

Reply:
Yes, this is the same as Joris's question.  The answer is that it is probably better to return EA-only, but ATM DTA/TDA/PA have existing implementations that return ndarray